index.html

```diff
@@ -213,8 +214,31 @@ <h1 class="big-heading">
                   Community of Empowered Women in Tech
                 </h1>
                 <h5 class="big-heading">
-                  Come be a part of a vibrant community of high achieving,
```

Comment:
Women not woman

Keep it original only no need to add any animation

Reply:
Just keep the first animation on this PR (for the line community of empowered...), no need to add any animation for second line. Keep it original

Reply:
Removed the animations from the second line, can you please check